tool_collections/taxonomy/kraken2tax/kraken2tax.xml

```diff
@@ -0,0 +1,102 @@
+<tool id="Kraken2Tax" name="Convert Kraken" version="1.0">
+  <description>data to Galaxy taxonomy representation</description>
+    <requirements>
+        <requirement type="package" version="8d245994d7">gb_taxonomy</requirement>
```

Comment:
@erasche @natefoo is awk a standard on all Galaxy supported system? Also syntax wise? Or do we want to depend on gnu_coreutils?